resources/views/suppliers/index.blade.php

```diff
@@ -0,0 +1,5 @@
+@extends('layouts.app')
+
+@section('content')
+<suppliers-index></suppliers-index>
+@endsection
```

Comment:
u can probs get rid of this and the controller since your using a vue for it in the router.js